tests/nnfw_api/src/CircleGen.h

```diff
@@ -54,6 +54,47 @@ class CircleGen
 public:
   using Shape = std::vector<int32_t>;
 
+  using SparseIndexVectorType = circle::SparseIndexVector;
+  using SparseDimensionType = circle::DimensionType;
+
+  struct SparseIndexVector
+  {
+    SparseIndexVector(){};
+    ~SparseIndexVector(){};
+    std::vector<uint16_t> u16;
+  };
```

Comment:
```suggestion
  struct SparseIndexVector
  {
    SparseIndexVector(){}
    ~SparseIndexVector(){}
    std::vector<uint16_t> u16;
  };
```
And maybe ctor and ctor could be removed as well. Is there any reason of this?